devops/roles/analytics-druid/defaults/main.yml

```diff
@@ -0,0 +1,129 @@
+---
+# Druid Template variables
+
+#Common Varibles
+
+druid_zookeeper_host: "172.17.0.4:2181"
+druid_postgres_host: "172.17.0.3"
+druid_postgres_user: "druid"
+#druid_postgres_pass: "secrets file"
+
+druid_default_tmp_dir: "/var/tmp"
+druid_gc_logdir: "/var/gclog"
+druid_log_dir: "/var/log/druid/"
+
+#azure config
+
+azure_account: "sunbirddevtelemetry"
+#azure_key: "secrets file"
```

Comment:
variablize the value, and move it to secrets.yml
azure_key: "{{ vault_azure_druid_key }}"
and the former should go to secrets.yml